maquetado/scss/login.scss

```diff
@@ -0,0 +1,76 @@
+.login-container {
+  background-color: $wild-sand;
+  display: flex;
+  flex-flow: column wrap;
+  height: 100%;
+  justify-content: center;
+  max-height: 400px;
+  max-width: 300px;
+  width: 100%;
+
+  .login-top {
+    background-color: $cerulean;
+    border: 2px solid $cerulean;
+    height: 100%;
+    max-height: 8px;
+    width: 100%;
```

Comment:
It is border this class. So you can add border: 10px.. to the container. Am I right?

Reply:
I'm sorry, but I can't follow you. Why would I want to do that?

Reply:
I get it now, Done!